python/pyspark/ml/clustering.py

```diff
@@ -873,6 +1049,31 @@ class LDAModel(JavaModel, _LDAParams):
     .. versionadded:: 2.0.0
     """
 
+    @since("3.0.0")
+    def setFeaturesCol(self, value):
+        """
+        Sets the value of :py:attr:`featuresCol`.
+        """
+        return self._set(featuresCol=value)
+
+    @since("3.0.0")
+    def setSeed(self, value):
+        """
+        Sets the value of :py:attr:`seed`.
+        """
+        return self._set(seed=value)
+
+    @since("3.0.0")
+    def setTopicDistributionCol(self, value):
+        """
+        Sets the value of :py:attr:`topicDistributionCol`.
+
+        >>> algo = LDA().setTopicDistributionCol("topicDistributionCol")
```

Comment:
If anything, shouldn't we have model specific dosctring here?  This one just duplicates what we already have in estimator:

https://github.com/apache/spark/blob/b389b8c5f0650a7e63098f18437fcaa29998732a/python/pyspark/ml/clustering.py#L1159-L1160

Reply:
I am in the middle of rechecking this PR and other ml parity PRs. I will have a follow up PR soon to fix this along with a few other problems I found.

Reply:
Thanks @huaxingao. If you don't mind me asking ‒ should I expect any serious shifts?

Reply:
No. Just fix the small things I overlooked before, for example, I have setters in ```TrainValidationSplitModel/CrossValidatorModel``` because these setters are used  in ```_from_java```.  I will remove the setters and use _set instead.

Reply:
Thanks @huaxingao.